src/cli/cli.cpp

```diff
@@ -602,6 +612,259 @@ void Interpreter::ProcessChildTimeout(int argc, char *argv[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_ENABLE_APPLICATION_COAP
+
+void Interpreter::ProcessCoapServer(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    VerifyOrExit(argc > 0, error = kThreadError_InvalidArgs);
+
+    if (strcmp(argv[0], "start") == 0)
+    {
+        SuccessOrExit(error = otCoapServerStart(mInstance));
+        SuccessOrExit(error = otCoapServerAddResource(mInstance, &mResource));
+        sServer->OutputFormat("Starting server and adding resource '%s' ... ", mResource.mUriPath);
+    }
+    else if (strcmp(argv[0], "stop") == 0)
+    {
+        otCoapServerRemoveResource(mInstance, &mResource);
+        SuccessOrExit(error = otCoapServerStop(mInstance));
+        sServer->OutputFormat("Stopping server ... ");
+    }
+    else if (strcmp(argv[0], "name") == 0)
+    {
+        if (argv[1]){
```

Comment:
Sure no problem!

```
$ ./bootstrap
$ ./configure --enable-ftd --enable-cli --enable-diag --enable-dhcp6-client --enable-dhcp6-server --enable-dns-client --enable-commissioner --enable-joiner --with-examples=posix
$ make pretty
```

If you are interest you can check how travis does all the checks in this file:  [script.sh](https://github.com/openthread/openthread/blob/master/.travis/script.sh)